Integrations/BluecatAddressManager/BluecatAddressManager.py

```diff
@@ -0,0 +1,390 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+''' IMPORTS '''
+
+import requests
+
+
+# error class for token errors
+class TokenException(Exception):
```

Comment:
where did this came from?

Reply:
Added this so I won't have to catch `Exception` in the `get_token` request executed in the beginning (this isn't necessary, so it can be removed)